qemu/tests/change_media.py

```diff
@@ -89,9 +91,6 @@ def get_qdev_by_filename(filename):
         msg = "Fail to insert device %s to guest" % orig_img_name
         test.fail(msg)
 
-    if check_block_locked(device_name):
-        test.fail("Unused device is locked.")
-
```

Comment:
why remove this?

Reply:
As we discussed last week, different qemu versions have different `locked` states, so I think we can remove this checkpoint.